Package.swift

```diff
@@ -16,19 +16,21 @@ let package = Package(
     name: "SpeziOnboarding",
     defaultLocalization: "en",
     platforms: [
-        .iOS(.v17),
-        .visionOS(.v1),
-        .macOS(.v14)
+        .iOS(.v18),
+        .visionOS(.v2),
+        .macOS(.v15)
     ],
     products: [
-        .library(name: "SpeziOnboarding", targets: ["SpeziOnboarding"])
+        .library(name: "SpeziOnboarding", targets: ["SpeziOnboarding"]),
+        .library(name: "SpeziConsent", targets: ["SpeziConsent"])
     ],
     dependencies: [
         .package(url: "https://github.com/StanfordSpezi/Spezi.git", from: "1.8.0"),
         .package(url: "https://github.com/StanfordSpezi/SpeziFoundation.git", from: "2.1.2"),
-        .package(url: "https://github.com/StanfordSpezi/SpeziViews.git", from: "1.10.0"),
+        .package(url: "https://github.com/StanfordSpezi/SpeziViews.git", branch: "lukas/CanvasView-disableDrawing"),
```

Comment:
TODO: change to tagged release once https://github.com/StanfordSpezi/SpeziViews/pull/65 is merged